// mdi-vue/DevTo.js
import { _ as __vue_normalize__ } from './normalize-component-26d614d1.js';

//
//
//
//
//
//
//
//
//
//
//
//
//
//

var script = {
  name: 'mdi-dev-to',
  props: {
    width: {
      type: [Number, String],
      default: 24
    },
    height: {
      type: [Number, String],
      default: 24
    },
    viewBox: {
      type: [String],
      default: '0 0 24 24'
    },
    xmlns: {
      type: String,
      default: 'xmlns="http://www.w3.org/2000/svg"'
    },
    ariaLabel: {
      type: String,
      default: 'DevTo icon'
    },
    role: {
      type: String,
      default: 'img'
    }
  }
};

/* script */
const __vue_script__ = script;

/* template */
var __vue_render__ = function(_h, _vm) {
  var _c = _vm._c;
  return _c(
    "span",
    {
      staticClass: "mdi mdi-dev-to",
      attrs: { role: _vm.props.role, "aria-label": _vm.props.ariaLabel }
    },
    [
      _c(
        "svg",
        {
          attrs: {
            fill: "currentColor",
            width: _vm.props.width,
            height: _vm.props.height,
            viewBox: _vm.props.viewBox,
            xmlns: _vm.props.xmlns
          }
        },
        [
          _c("path", {
            attrs: {
              d:
                "M7.36,9.87C7.19,9.74 7,9.68 6.84,9.68H6.06V14.34H6.84C7,14.34 7.19,14.28 7.36,14.15C7.54,14 7.62,13.82 7.62,13.56V10.45C7.62,10.19 7.54,10 7.36,9.87M20.04,2H3.96A1.96,1.96 0 0,0 2,3.96V20.04C2,21.13 2.88,22 3.96,22H20.04A1.96,1.96 0 0,0 22,20.04V3.96C22,2.87 21.12,2 20.04,2M8.88,13.57C8.88,14.41 8.37,15.68 6.73,15.68H4.65V8.29H6.77C8.35,8.29 8.88,9.56 8.88,10.4V13.57M13.38,9.61H11V11.33H12.45V12.65H11V14.36H13.38V15.68H10.6C10.1,15.7 9.69,15.3 9.68,14.8V9.22C9.67,8.72 10.06,8.31 10.56,8.3H13.38V9.61M18,14.76C17.42,16.13 16.36,15.86 15.89,14.76L14.17,8.3H15.62L16.95,13.37L18.27,8.3H19.72L18,14.76V14.76Z"
            }
          })
        ]
      )
    ]
  )
};
var __vue_staticRenderFns__ = [];
__vue_render__._withStripped = true;

  /* style */
  const __vue_inject_styles__ = undefined;
  /* scoped */
  const __vue_scope_id__ = undefined;
  /* module identifier */
  const __vue_module_identifier__ = undefined;
  /* functional template */
  const __vue_is_functional_template__ = true;
  /* style inject */
  
  /* style inject SSR */
  

  
  var DevTo = __vue_normalize__(
    { render: __vue_render__, staticRenderFns: __vue_staticRenderFns__ },
    __vue_inject_styles__,
    __vue_script__,
    __vue_scope_id__,
    __vue_is_functional_template__,
    __vue_module_identifier__,
    undefined,
    undefined
  );

export default DevTo;
