// mdi-vue/DiceD10.js
import { _ as __vue_normalize__ } from './normalize-component-26d614d1.js';

//
//
//
//
//
//
//
//
//
//
//
//
//
//

var script = {
  name: 'mdi-dice-d-10',
  props: {
    width: {
      type: [Number, String],
      default: 24
    },
    height: {
      type: [Number, String],
      default: 24
    },
    viewBox: {
      type: [String],
      default: '0 0 24 24'
    },
    xmlns: {
      type: String,
      default: 'xmlns="http://www.w3.org/2000/svg"'
    },
    ariaLabel: {
      type: String,
      default: 'DiceD10 icon'
    },
    role: {
      type: String,
      default: 'img'
    }
  }
};

/* script */
const __vue_script__ = script;

/* template */
var __vue_render__ = function(_h, _vm) {
  var _c = _vm._c;
  return _c(
    "span",
    {
      staticClass: "mdi mdi-dice-d-10",
      attrs: { role: _vm.props.role, "aria-label": _vm.props.ariaLabel }
    },
    [
      _c(
        "svg",
        {
          attrs: {
            fill: "currentColor",
            width: _vm.props.width,
            height: _vm.props.height,
            viewBox: _vm.props.viewBox,
            xmlns: _vm.props.xmlns
          }
        },
        [
          _c("path", {
            attrs: {
              d:
                "M15 10.5V13.27C15 13.85 14.5 14.33 13.92 14.33C13.34 14.33 12.86 13.86 12.86 13.27V10.5C12.86 9.91 13.34 9.44 13.92 9.44C14.5 9.44 15 9.91 15 10.5M21.5 13.2L13.2 21.5C12.5 22.2 11.5 22.2 10.8 21.5L2.5 13.2C1.8 12.5 1.8 11.5 2.5 10.8L10.8 2.5C11.5 1.8 12.5 1.8 13.2 2.5L21.5 10.8C22.1 11.5 22.1 12.5 21.5 13.2M10.38 8.2H10.22L7.08 9.32V10.55L8.88 10V15.79H10.38V8.2M16.5 10.57C16.5 9.15 15.35 8 13.93 8S11.36 9.15 11.36 10.57V13.21C11.36 14.63 12.5 15.78 13.93 15.78C15.35 15.78 16.5 14.63 16.5 13.21V10.57Z"
            }
          })
        ]
      )
    ]
  )
};
var __vue_staticRenderFns__ = [];
__vue_render__._withStripped = true;

  /* style */
  const __vue_inject_styles__ = undefined;
  /* scoped */
  const __vue_scope_id__ = undefined;
  /* module identifier */
  const __vue_module_identifier__ = undefined;
  /* functional template */
  const __vue_is_functional_template__ = true;
  /* style inject */
  
  /* style inject SSR */
  

  
  var DiceD10 = __vue_normalize__(
    { render: __vue_render__, staticRenderFns: __vue_staticRenderFns__ },
    __vue_inject_styles__,
    __vue_script__,
    __vue_scope_id__,
    __vue_is_functional_template__,
    __vue_module_identifier__,
    undefined,
    undefined
  );

export default DiceD10;
