// mdi-vue/EmailVariant.js
import { _ as __vue_normalize__ } from './normalize-component-26d614d1.js';

//
//
//
//
//
//
//
//
//
//
//
//
//
//

var script = {
  name: 'mdi-email-variant',
  props: {
    width: {
      type: [Number, String],
      default: 24
    },
    height: {
      type: [Number, String],
      default: 24
    },
    viewBox: {
      type: [String],
      default: '0 0 24 24'
    },
    xmlns: {
      type: String,
      default: 'xmlns="http://www.w3.org/2000/svg"'
    },
    ariaLabel: {
      type: String,
      default: 'EmailVariant icon'
    },
    role: {
      type: String,
      default: 'img'
    }
  }
};

/* script */
const __vue_script__ = script;

/* template */
var __vue_render__ = function(_h, _vm) {
  var _c = _vm._c;
  return _c(
    "span",
    {
      staticClass: "mdi mdi-email-variant",
      attrs: { role: _vm.props.role, "aria-label": _vm.props.ariaLabel }
    },
    [
      _c(
        "svg",
        {
          attrs: {
            fill: "currentColor",
            width: _vm.props.width,
            height: _vm.props.height,
            viewBox: _vm.props.viewBox,
            xmlns: _vm.props.xmlns
          }
        },
        [
          _c("path", {
            attrs: {
              d:
                "M12,13L2,6.76V6C2,4.89 2.89,4 4,4H20A2,2 0 0,1 22,6V6.75L12,13M22,18A2,2 0 0,1 20,20H4C2.89,20 2,19.1 2,18V9.11L4,10.36V18H20V10.36L22,9.11V18Z"
            }
          })
        ]
      )
    ]
  )
};
var __vue_staticRenderFns__ = [];
__vue_render__._withStripped = true;

  /* style */
  const __vue_inject_styles__ = undefined;
  /* scoped */
  const __vue_scope_id__ = undefined;
  /* module identifier */
  const __vue_module_identifier__ = undefined;
  /* functional template */
  const __vue_is_functional_template__ = true;
  /* style inject */
  
  /* style inject SSR */
  

  
  var EmailVariant = __vue_normalize__(
    { render: __vue_render__, staticRenderFns: __vue_staticRenderFns__ },
    __vue_inject_styles__,
    __vue_script__,
    __vue_scope_id__,
    __vue_is_functional_template__,
    __vue_module_identifier__,
    undefined,
    undefined
  );

export default EmailVariant;
