// mdi-vue/Laravel.js
import { _ as __vue_normalize__ } from './normalize-component-26d614d1.js';

//
//
//
//
//
//
//
//
//
//
//
//
//
//

var script = {
  name: 'mdi-laravel',
  props: {
    width: {
      type: [Number, String],
      default: 24
    },
    height: {
      type: [Number, String],
      default: 24
    },
    viewBox: {
      type: [String],
      default: '0 0 24 24'
    },
    xmlns: {
      type: String,
      default: 'xmlns="http://www.w3.org/2000/svg"'
    },
    ariaLabel: {
      type: String,
      default: 'Laravel icon'
    },
    role: {
      type: String,
      default: 'img'
    }
  }
};

/* script */
const __vue_script__ = script;

/* template */
var __vue_render__ = function(_h, _vm) {
  var _c = _vm._c;
  return _c(
    "span",
    {
      staticClass: "mdi mdi-laravel",
      attrs: { role: _vm.props.role, "aria-label": _vm.props.ariaLabel }
    },
    [
      _c(
        "svg",
        {
          attrs: {
            fill: "currentColor",
            width: _vm.props.width,
            height: _vm.props.height,
            viewBox: _vm.props.viewBox,
            xmlns: _vm.props.xmlns
          }
        },
        [
          _c("path", {
            attrs: {
              d:
                "M22,11.87C22.03,11.66 21.96,11.43 21.81,11.27C21.75,11.2 21.29,10.63 20.81,10.03C20.32,9.42 19.8,8.77 19.64,8.59C19.46,8.37 19.2,8.12 18.81,8.12C18.74,8.12 18.68,8.13 18.58,8.14C18.46,8.15 17.53,8.31 16.8,8.43C16.36,8.5 16,8.56 15.9,8.58C15.55,8.63 15.32,8.78 15.22,9C15.17,9.14 15.11,9.4 15.33,9.69L16.83,11.82L12.08,12.95L7.63,5.5C7.5,5.29 7.29,5 6.76,5C6.71,5 6.66,5 6.61,5L5.59,5.1C4.68,5.17 3,5.31 2.82,5.31C2.5,5.31 2.27,5.43 2.13,5.65C1.96,5.9 1.96,6.23 2.13,6.63C2.37,7.19 6.04,15.12 6.06,15.16C6.25,15.64 6.71,15.95 7.22,15.95C7.36,15.95 7.5,15.93 7.59,15.89C8.16,15.75 10,15.27 11.24,14.95C11.87,16.09 13,18.12 13.24,18.45C13.4,18.68 13.67,19 14.14,19C14.35,19 14.55,18.93 14.79,18.85C15.14,18.74 20.74,16.74 21.03,16.62C21.23,16.53 21.5,16.42 21.6,16.11C21.66,15.91 21.62,15.7 21.47,15.5C21.36,15.32 20.31,13.92 19.53,12.88L21.47,12.36C21.82,12.25 21.94,12.03 22,11.87M13,14.5L17.89,13.24C18.73,14.39 19.18,15 19.42,15.33C15.88,16.53 14.79,16.9 14.47,17C14.24,16.65 13.56,15.47 13,14.5M17.3,9.87C17.76,9.79 18.35,9.68 18.67,9.62C18.88,9.89 19.44,10.6 19.83,11.1C19.4,11.21 18.8,11.36 18.44,11.45L17.3,9.87M6.38,6.5L10.35,13.38L7.6,14.04C7,12.79 4.78,8.22 4.06,6.72C4.32,6.7 6.13,6.54 6.38,6.5Z"
            }
          })
        ]
      )
    ]
  )
};
var __vue_staticRenderFns__ = [];
__vue_render__._withStripped = true;

  /* style */
  const __vue_inject_styles__ = undefined;
  /* scoped */
  const __vue_scope_id__ = undefined;
  /* module identifier */
  const __vue_module_identifier__ = undefined;
  /* functional template */
  const __vue_is_functional_template__ = true;
  /* style inject */
  
  /* style inject SSR */
  

  
  var Laravel = __vue_normalize__(
    { render: __vue_render__, staticRenderFns: __vue_staticRenderFns__ },
    __vue_inject_styles__,
    __vue_script__,
    __vue_scope_id__,
    __vue_is_functional_template__,
    __vue_module_identifier__,
    undefined,
    undefined
  );

export default Laravel;
