// mdi-vue/ScrewRoundTop.js
import { _ as __vue_normalize__ } from './normalize-component-26d614d1.js';

//
//
//
//
//
//
//
//
//
//
//
//
//
//

var script = {
  name: 'mdi-screw-round-top',
  props: {
    width: {
      type: [Number, String],
      default: 24
    },
    height: {
      type: [Number, String],
      default: 24
    },
    viewBox: {
      type: [String],
      default: '0 0 24 24'
    },
    xmlns: {
      type: String,
      default: 'xmlns="http://www.w3.org/2000/svg"'
    },
    ariaLabel: {
      type: String,
      default: 'ScrewRoundTop icon'
    },
    role: {
      type: String,
      default: 'img'
    }
  }
};

/* script */
const __vue_script__ = script;

/* template */
var __vue_render__ = function(_h, _vm) {
  var _c = _vm._c;
  return _c(
    "span",
    {
      staticClass: "mdi mdi-screw-round-top",
      attrs: { role: _vm.props.role, "aria-label": _vm.props.ariaLabel }
    },
    [
      _c(
        "svg",
        {
          attrs: {
            fill: "currentColor",
            width: _vm.props.width,
            height: _vm.props.height,
            viewBox: _vm.props.viewBox,
            xmlns: _vm.props.xmlns
          }
        },
        [
          _c("path", {
            attrs: {
              d:
                "M13.5,17V19L12,22L10.5,19L13.5,17M14.5,6.3L13.5,7V6H10.5V9L9.5,9.7V10.7L14.5,7.4V6.3M14.5,10.3L13.5,11V9L10.5,11V13L9.5,13.7V14.7L14.5,11.4V10.3M14.5,14.3L13.5,15V13L10.5,15V17L9.5,17.7V18.7L14.5,15.4V14.3M7,5H17C17,5 16,2 12,2C8,2 7,5 7,5Z"
            }
          })
        ]
      )
    ]
  )
};
var __vue_staticRenderFns__ = [];
__vue_render__._withStripped = true;

  /* style */
  const __vue_inject_styles__ = undefined;
  /* scoped */
  const __vue_scope_id__ = undefined;
  /* module identifier */
  const __vue_module_identifier__ = undefined;
  /* functional template */
  const __vue_is_functional_template__ = true;
  /* style inject */
  
  /* style inject SSR */
  

  
  var ScrewRoundTop = __vue_normalize__(
    { render: __vue_render__, staticRenderFns: __vue_staticRenderFns__ },
    __vue_inject_styles__,
    __vue_script__,
    __vue_scope_id__,
    __vue_is_functional_template__,
    __vue_module_identifier__,
    undefined,
    undefined
  );

export default ScrewRoundTop;
