// mdi-vue/TrophyBroken.js
import { _ as __vue_normalize__ } from './normalize-component-26d614d1.js';

//
//
//
//
//
//
//
//
//
//
//
//
//
//

var script = {
  name: 'mdi-trophy-broken',
  props: {
    width: {
      type: [Number, String],
      default: 24
    },
    height: {
      type: [Number, String],
      default: 24
    },
    viewBox: {
      type: [String],
      default: '0 0 24 24'
    },
    xmlns: {
      type: String,
      default: 'xmlns="http://www.w3.org/2000/svg"'
    },
    ariaLabel: {
      type: String,
      default: 'TrophyBroken icon'
    },
    role: {
      type: String,
      default: 'img'
    }
  }
};

/* script */
const __vue_script__ = script;

/* template */
var __vue_render__ = function(_h, _vm) {
  var _c = _vm._c;
  return _c(
    "span",
    {
      staticClass: "mdi mdi-trophy-broken",
      attrs: { role: _vm.props.role, "aria-label": _vm.props.ariaLabel }
    },
    [
      _c(
        "svg",
        {
          attrs: {
            fill: "currentColor",
            width: _vm.props.width,
            height: _vm.props.height,
            viewBox: _vm.props.viewBox,
            xmlns: _vm.props.xmlns
          }
        },
        [
          _c("path", {
            attrs: {
              d:
                "M2,2V11C2,12 3,13 4,13H4.5L6.2,13C6.6,14.8 7.6,16.3 10,16.8V19.2C8.6,19.6 8,20.5 8,21.6V22H13.2L10.2,15L14.2,10L10.7,4H8V3.7C7.8,2.8 6.8,2 6,2H4.5L3.8,2H2M13.2,22H16V21.7C16,21.1 15.8,20.5 15.4,20.1C15.1,19.8 14.6,19.5 14,19.3V16.9C16.4,16.4 17.5,14.8 17.8,13.1H20C21,13.1 22,12.1 22,11.1V5.7L22,2H20.2L19.5,2H18C17.2,2 16.3,2.7 16.1,3.5C16,3.7 16,3.8 16,4H13.9L16.7,10L12.2,15L13.2,22M4,4H6V4.4L6,6V11H4V6.4L4,4M18,4H19.7L20,4V4.1L20,11H18V6L18,4Z"
            }
          })
        ]
      )
    ]
  )
};
var __vue_staticRenderFns__ = [];
__vue_render__._withStripped = true;

  /* style */
  const __vue_inject_styles__ = undefined;
  /* scoped */
  const __vue_scope_id__ = undefined;
  /* module identifier */
  const __vue_module_identifier__ = undefined;
  /* functional template */
  const __vue_is_functional_template__ = true;
  /* style inject */
  
  /* style inject SSR */
  

  
  var TrophyBroken = __vue_normalize__(
    { render: __vue_render__, staticRenderFns: __vue_staticRenderFns__ },
    __vue_inject_styles__,
    __vue_script__,
    __vue_scope_id__,
    __vue_is_functional_template__,
    __vue_module_identifier__,
    undefined,
    undefined
  );

export default TrophyBroken;
