// mdi-vue/Yahoo.js
import { _ as __vue_normalize__ } from './normalize-component-26d614d1.js';

//
//
//
//
//
//
//
//
//
//
//
//
//
//

var script = {
  name: 'mdi-yahoo',
  props: {
    width: {
      type: [Number, String],
      default: 24
    },
    height: {
      type: [Number, String],
      default: 24
    },
    viewBox: {
      type: [String],
      default: '0 0 24 24'
    },
    xmlns: {
      type: String,
      default: 'xmlns="http://www.w3.org/2000/svg"'
    },
    ariaLabel: {
      type: String,
      default: 'Yahoo icon'
    },
    role: {
      type: String,
      default: 'img'
    }
  }
};

/* script */
const __vue_script__ = script;

/* template */
var __vue_render__ = function(_h, _vm) {
  var _c = _vm._c;
  return _c(
    "span",
    {
      staticClass: "mdi mdi-yahoo",
      attrs: { role: _vm.props.role, "aria-label": _vm.props.ariaLabel }
    },
    [
      _c(
        "svg",
        {
          attrs: {
            fill: "currentColor",
            width: _vm.props.width,
            height: _vm.props.height,
            viewBox: _vm.props.viewBox,
            xmlns: _vm.props.xmlns
          }
        },
        [
          _c("path", {
            attrs: {
              d:
                "M12.03,3.36C8.5,3.36 5.15,2.9 2,2C2,9.07 2,20.23 2,22C5.15,21.1 8.5,20.64 12.03,20.64C15.5,20.64 18.85,21.09 22,22C22,15.2 22,9.31 22,2C18.85,2.91 15.5,3.36 12.03,3.36M12.84,13.04L12.97,18.92L12,18.79C12,18.79 12,18.79 12,18.79C11.91,18.79 11.19,18.89 11,18.93L11.16,13.04C10.88,12.54 6.67,5.44 6.44,5.07L7.5,5.25C7.83,5.25 8.12,5.21 8.5,5.17C9.03,6.15 11.88,10.83 12,11.03C12,11.03 14.97,6.15 15.61,5.13C15.88,5.21 16.17,5.25 16.5,5.25C16.78,5.25 17.07,5.21 17.57,5.07L12.84,13.04Z"
            }
          })
        ]
      )
    ]
  )
};
var __vue_staticRenderFns__ = [];
__vue_render__._withStripped = true;

  /* style */
  const __vue_inject_styles__ = undefined;
  /* scoped */
  const __vue_scope_id__ = undefined;
  /* module identifier */
  const __vue_module_identifier__ = undefined;
  /* functional template */
  const __vue_is_functional_template__ = true;
  /* style inject */
  
  /* style inject SSR */
  

  
  var Yahoo = __vue_normalize__(
    { render: __vue_render__, staticRenderFns: __vue_staticRenderFns__ },
    __vue_inject_styles__,
    __vue_script__,
    __vue_scope_id__,
    __vue_is_functional_template__,
    __vue_module_identifier__,
    undefined,
    undefined
  );

export default Yahoo;
